src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibratorIntegrationTest.java

```diff
@@ -26,6 +26,38 @@
  */
 public class VariantRecalibratorIntegrationTest extends CommandLineProgramTest {
 
+    private final String[] VQSRParamsWithResources =
+        new String[] {
+            "--variant",
+            getLargeVQSRTestDataDir() + "phase1.projectConsensus.chr20.1M-10M.raw.snps.vcf",
+            "-L","20:1,000,000-10,000,000",
+            "--resource:known,known=true,prior=10.0",
+            getLargeVQSRTestDataDir() + "dbsnp_132_b37.leftAligned.20.1M-10M.vcf",
+            "--resource:truth_training1,truth=true,training=true,prior=15.0",
+            getLargeVQSRTestDataDir() + "sites_r27_nr.b37_fwd.20.1M-10M.vcf",
+            "--resource:truth_training2,training=true,truth=true,prior=12.0",
+            getLargeVQSRTestDataDir() + "Omni25_sites_1525_samples.b37.20.1M-10M.vcf",
+            "-an", "QD", "-an", "HaplotypeScore", "-an", "HRun",
+            "--trust-all-polymorphic", // for speed
+            "-mode", "SNP",
+            "--" + StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, "false"
+        };
+
+    private final String[] alleleSpecificVQSRParams =
+        new String[] {
+            "--variant",
+            getLargeVQSRTestDataDir() + "chr1snippet.doctoredMQ.sites_only.vcf.gz",
+            "-L","chr1:1-10,000,000",
+            "-resource:same,known=false,training=true,truth=true,prior=15",
+            getLargeVQSRTestDataDir() + "chr1snippet.doctoredMQ.sites_only.vcf.gz",
+            "-an", "AS_QD", "-an", "AS_ReadPosRankSum", "-an", "AS_MQ", "-an", "AS_SOR", //AS_MQRankSum has zero variance and AS_FS is nearly constant; also different annotation orders may not converge
+            "--trust-all-polymorphic", // for speed
+            "--use-allele-specific-annotations",
+            "-mode", "SNP",
+            "--" + StandardArgumentDefinitions.ADD_OUTPUT_VCF_COMMANDLINE, "false",
+            "--max-gaussians", "6"
```

Comment:
Presumably this test fails if the number of gaussians is not reduced. Can you add this same test case, but without this arg, as a negative test case, since I don't think we actually have any of those.

Reply:
It turns out it doesn't fail, so I took the arg out and updated the results.  Added two different failing tests.